### DISCUSSION TITLE
Missing button icon

### DISCUSSION THREAD

**Discussion by StyingDev** [amnweb/yasb#38] [Q&A] [answered]
Hello, I seem to have the button icons not be loaded
even though i have nerd font installed (jettbrains one)
![image](https://github.com/user-attachments/assets/40c32709-722e-41e3-9985-a6f7fe89f86e)

**Comment by amnweb** [comment]
This screenshot says that the font is missing. Try to reboot the PC to check if the font will be loaded.

**Reply by StyingDev** [reply]
I tried, doesnt seem to work
could i replace the icons with something else maybe? 
thanks.

**Reply by amnweb** [reply]
Can you try load default config from repo ?

**Comment by StyingDev** [comment, marked as answer]
@amnweb I have fixed it,

Apparently the issue was that the icon that was used on default for YASB the media player icon  "Segoe Fluent Icons" is not installed on windows 10 by default however it was on windows 11. So, you have to manually install "Segoe Fluent Icons" from [Here](https://learn.microsoft.com/en-us/windows/apps/design/downloads/#fonts) on windows 10. That seemed to get my issue fixed after installing it and restarting.

I was hoping if you could put a notice in the readme for windows 10 user perhaps, thanks.

**Reply by amnweb** [reply]
I didn't know that Windows 10 does not have this fonts installed, thank for info.
